web/libs/editor/src/components/Entities/Entities.jsx

```diff
@@ -0,0 +1,113 @@
+import { Dropdown } from "antd";
```

Comment:
this is likely copied in from another spot - but antd instead of our Dropdown? 😢